instrumentation/opencensus-shim/testing/src/test/java/io/opentelemetry/opencensusshim/SpanConverterProxy.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.opencensusshim;
+
+import io.opentelemetry.api.trace.SpanContext;
+
+public class SpanConverterProxy {
+
+  private SpanConverterProxy() {}
+
+  public static SpanContext mapSpanContext(io.opencensus.trace.SpanContext ctx) {
+    return SpanConverter.mapSpanContext(ctx);
```

Comment:
is it worth just inlining this and removing the class?

Reply:
I'd agree but it can't be inlined because `SpanConverter` is [package-private](https://github.com/open-telemetry/opentelemetry-java/blob/main/opencensus-shim/src/main/java/io/opentelemetry/opencensusshim/SpanConverter.java#L23).

Reply:
can you add a short class level comment to explain why it's needed?

Reply:
Done :+1: